docs/source/changelog.rst

```diff
@@ -7,13 +7,14 @@ Changelog
     * Fixes
         * Improve warnings when using a Dask dataframe for cutoff times (:pr:`1026`)
     * Changes
+        * Deprecated ``Feature.get_names()``, ``Feature.get_feature_names()`` should be used instead(:pr:`1021`)
```

Comment:
was `get_names()` removed? if so, we should say "removed" not "deprecated". Deprecate means that we're in the process of removing want to tell people to stop using it, rather than have actually finished removing. See definition [here](https://techterms.com/definition/deprecated#:~:text=In%20the%20world%20of%20software,means%20to%20disapprove%20of%20something.)

Since we are removing, we should also add this to a breaking changes section with the comment of what to do instead.